portfolio/src/main/webapp/index.html

```diff
@@ -13,6 +13,29 @@ <h1>My Portfolio</h1>
       <p>Click here to get my name:</p>
       <button onclick="getRandomMyNameUsingAsyncAwait()">Hello</button>
       <div id="name-container"></div>
+
+      <h1>Server Stats</h1>
+      <p>Click here to see JSON string:</p>
+      <button onclick="getJSONString()">Fetch JSON string!</button>
+      <ul id="json-string"></ul>
+
+      <h1>Modify your word</h1>
+      <form action="/data" method="POST">
+
+        <p>Do you like my portfolio?</p>
+        <textarea name="word-input">Input feedback here.</textarea>
+        <br/><br/>
+
+        <p>Choose some options:</p>
+        <input type="checkbox" name="positive" value="true"> Yes, I like it!
```

Comment:
These three options look like they're meant to be mutually exclusive. Thoughts on using radio buttons rather than checkboxes?

Reply:
That sounds like a great idea, just did the changes that you discussed above as well! What did you mean before by the single enum vs the booleans? (just curious!)

Reply:
Since the values of "positive", "negative", and "mixed" are mutually exclusive it doesn't make a lot of sense to store each one separately as a boolean. It makes more sense to store a single enum that can be "Positive", "Negative", or "Mixed". This not only prevents you from getting into a state where something is both positive and negative but also makes adding and removing options much easier.

Reply:
Yes, that makes a lot of sense. When you mean single enum, you mean single number?

Reply:
Behind the scenes enums are usually numbers but the point is to obscure that so they're human readable.
https://www.w3schools.com/java/java_enums.asp
https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/